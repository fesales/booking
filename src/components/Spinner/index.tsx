import React from 'react';
import './index.css';

const Spinner: React.FunctionComponent = () => {
  return <div className="spinner" />;
};

export default Spinner;
