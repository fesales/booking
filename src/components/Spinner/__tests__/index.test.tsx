import React from 'react';
import { shallow } from 'enzyme';
import Spinner from '..';

describe('Spinner', () => {
  it('should render component', () => {
    const wrapper = shallow(<Spinner />);

    expect(wrapper).toMatchSnapshot();
    expect(wrapper.find('.spinner')).toHaveLength(1);
  });
});
