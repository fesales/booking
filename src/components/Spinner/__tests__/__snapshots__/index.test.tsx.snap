// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Spinner should render component 1`] = `
<div
  className="spinner"
/>
`;
